problem_236_a/Problem.java

```diff
@@ -1,5 +1,22 @@
+import java.util.Scanner;
+import java.util.HashSet;
+
 public class Problem {
   public static void main(String[] a) {
-    
+    Scanner sc = new Scanner(System.in);
+    String input = sc.nextLine();
+    HashSet<Character> inputChars = new HashSet<>();
```

Comment:
When you work with collections it is best practice to use interface, not exact implementation.
Here it should be 
```
Set<Character> ...
```

But as we discussed in https://github.com/raqimbek/code_challenges_and_problems/pull/3/files#r1828890041 , you can use keyword "var", but you must provide good name which will describe what variable contains.
For example, here I will give the name like "uniqueCharsSet"